### PR TITLE
Fix create Private Key on bitbucket.

### PR DESCRIPTION
**Pull request by appleboy**
cc @jasmo

### CODE REVIEW

File: content/installation/services/bitbucket_server.md [hunk 1]
```diff
@@ -40,13 +40,13 @@ NAME                        | DESC
 The OAuth process in Bitbucket server requires a private and a public RSA certificate. This is how you create the private RSA certificate.
 
 ```
-openssl genrsa -out mykey.pem 1024
+openssl genrsa -out /etc/bitbucket/key.pem 1024
 ```
 
 This stores the private RSA certificate in `mykey.pem`. The next command generates the public RSA certificate and stores it in `key.pem`.
```
Comment (gtaylor) [inline content/installation/services/bitbucket_server.md]: It looks like this should be `key.pem` and `key.pub` now.

**Comment by gtaylor** [comment]
Just the one nit

File: content/installation/services/bitbucket_server.md (hunk 1 above)
Comment (appleboy) [inline content/installation/services/bitbucket_server.md]: @gtaylor Done!

**Comment by gtaylor** [comment]
Thanks!

**Pull request merged**